_includes/form.html

```diff
@@ -0,0 +1,656 @@
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Enter Event Details</title>
+  <link rel="stylesheet" style="text/css" href="/json_form/deps/opt/bootstrap.css" data-proofer-ignore />
```

Comment:
Let's just do like this so we don't have to host these files ourselves:

https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css

https://www.w3schools.com/bootstrap/tryit.asp?filename=trybs_default&stacked=h